hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -809,7 +786,6 @@
       tests: ["chip_sw_clkmgr_external_clk_src_for_sw_fast",
               "chip_sw_clkmgr_external_clk_src_for_sw_slow",
               "chip_sw_clkmgr_external_clk_src_for_lc"]
-      tags: ["conn"]
```

Comment:
there are sim tests associate with it, so I think it is not a conn test?

Reply:
hm it looks like the naming is a bit confusing here. according to the description of the test on https://docs.opentitan.org/hw/top_earlgrey/doc/dv/ it looks like this is supposed to be both a connectivity test and simulation based tests. what is the procedure in this case? do we need to make a separate test point for the connectivity part?

Reply:
Yes I think we should have a separate testpoint for connectivity, otherwise it won't map to nightly regression result

Reply:
Good, can you take care of this in this PR also (sorry if you already did I missed it)?

It probably makes sense to rename the simulation test point to something like `chip_clkmgr_div` here and in the spreadsheet to avoid confusion.

Reply:
Thank you, the connectivity testplan already documented the clock divisions. 
1). I will re-name the simulation test point in this PR.
2). I will also update the testpoint in the spreadsheet.

Reply:
Thanks!